x/ibc-transfer/client/cli/tx.go

```diff
@@ -72,13 +77,16 @@ to the counterparty channel. Any timeout set to 0 is disabled.`),
 			// if the timeouts are not absolute, retrieve latest block height and block timestamp
 			// for the consensus state connected to the destination port/channel
 			if !absoluteTimeouts {
-				consensusState, _, err := channelutils.QueryCounterpartyConsensusState(clientCtx, srcPort, srcChannel, uint64(clientCtx.Height))
+				consensusState, _, err := channelutils.QueryLatestConsensusState(clientCtx, srcPort, srcChannel)
```

Comment:
This was incorrect, we cannot query consensus state with a consensus height using our own block height